pom.xml

```diff
@@ -9,7 +9,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>de.tum.in.www1</groupId>
     <artifactId>bamboo-server</artifactId>
-    <version>1.2.3</version>
+    <version>1.3</version>
```

Comment:
I believe we always used versions like `1.2.0` before, so we might want to go with `1.3.0` instead.